frontend/src/services/AuthService.ts

```diff
@@ -1,14 +1,24 @@
-import { AuthApi, AuthControllerSignUpUserRequest } from './api/apis/AuthApi';
+import { AuthApi, AuthControllerInducteeSignUpRequest } from './api/apis/AuthApi';
 import { AppUserSignupRequest, AppUserResponse } from './api/models';
 import ApiConfigStore from './ApiConfigStore';
 import { Configuration } from './api/runtime';
 
-export async function createNewUser(
+export async function createNewInducteeUser(
+  appUserSignupRequest: AppUserSignupRequest
+): Promise<AppUserResponse> {
+  const apiConfig: Configuration = await ApiConfigStore.getApiConfig();
+  const authApi: AuthApi = new AuthApi(apiConfig);
+  const request: AuthControllerInducteeSignUpRequest = { appUserSignupRequest };
+
+  return authApi.authControllerInducteeSignUp(request);
+}
+
+/*export async function createNewUser(
```

Comment:
also why is it commented?

Reply:
Simply because the old signup workflow used this. Reasoning is the same as my response above.